### PR TITLE
Iss augmented exercise

### PR DESCRIPTION
**Pull request by noratop**
I tried to figure out what should happens if the ISS request would take so long that the program would not be able to calculate the distance. Since there's no guarantee we get the ISS position, should we just tell that the calculation is not possible? same question applies for the request on the user location.

### CODE REVIEW

**Comment by noratop** [comment]
Should I put the request inside the other one ?

**Comment by noratop** [comment]
Ok I guess I found the solution to my problem above.

File: iss.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                    issLatLong.lng = issLatLong.iss_position.longitude;
+                        
+                    var distance = distanceTo(userGeoLocation,issLatLong);
+                    
+                    console.log('You are located '+ Math.round(distance) + 'km from the ISS station');
+                } 
+                else console.log('An error has occured.')
+            });
+        }
+        else console.log('An error has occured.')
```
Comment (nyanofthemoon) [inline iss.js]: It's good practice to let the end user know what error has occurred.

**Comment by ziad-saab** [comment]
perfect :100:

**Pull request closed without merging**